install/battlesnake.yml

```diff
@@ -0,0 +1,15 @@
+id: battlesnake
+
+name: Custom Attributes
+
+title: APM: Report custom events and attributes
```

Comment:
The `APM:` here is causing yaml parsing errors. Moving it to a piped field should alleviate the issue
```suggestion
title: |
  APM: Report custom events and attributes
```

Reply:
Hmm, that syntax highlighting has me worried